### PR TITLE
New release process and snapshot publishing

### PR DESCRIPTION
**Pull request by JakeWharton**
Based on the same thing we use everywhere else.

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,3 +1,21 @@
+GROUP=app.cash.duktape
```
Comment (mattprecious) [inline gradle.properties:1]: Intentional change?
```suggestion
GROUP=com.squareup.duktape
```

**Review by mattprecious — APPROVED** [review]

File: gradle.properties (hunk 1 above)
Comment (JakeWharton) [reply gradle.properties:1]: Hmm yeah let's leave duktape in the old and quickjs in the new

File: gradle.properties [hunk 2]
```diff
@@ -1,3 +1,21 @@
...
+POM_URL=https://github.com/cashapp/duktape-android/
+POM_SCM_URL=https://github.com/cashapp/duktape-android/
+POM_SCM_CONNECTION=scm:git:git://github.com/cashapp/duktape-android.git
+POM_SCM_DEV_CONNECTION=scm:git:ssh://git@github.com/cashapp/duktape-android.git
+
+POM_LICENCE_NAME=Apache-2.0
+POM_LICENCE_URL=https://www.apache.org/licenses/LICENSE-2.0
+POM_LICENCE_DIST=repo
+
+POM_DEVELOPER_ID=cashapp
```
Comment (mattprecious) [inline gradle.properties:15]: All this has changed, too. Intentional?

Comment (JakeWharton) [reply gradle.properties:15]: Yeah I mean the repo moved and this is what we use on other cashapp projects. The values are mostly irrelevant but are required.

**Pull request merged**